lease/lessor.go

```diff
@@ -161,13 +168,24 @@ func (le *lessor) Attach(id LeaseID, items []leaseItem) error {
 	return nil
 }
 
+func (le *Lessor) Recover(b backend.Backend, dr DeleteableRange) {
```

Comment:
@heyitsanthony We do not want to swap the pointer. The lessor pointer might be hold in several pkgs (server pkg, api pkg) inside etcdserver.